### ISSUE TITLE
1.5.0 appimage cannot locate legendary 

### ISSUE THREAD

**Issue by stephendcofer** [flavioislima/HeroicGamesLauncher#168]
```
$ ./heroic-1.5.0.AppImage
Error occurred in handler for 'writeLibrary': Error: Command failed: legendary list-games --json
/bin/sh: 1: legendary: not found

    at ChildProcess.exithandler (child_process.js:312:12)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:443:11)
    at Socket.emit (events.js:315:20)
    at Pipe.<anonymous> (net.js:674:12) {
  killed: false,
  code: 127,
  signal: null,
  cmd: 'legendary list-games --json',
  stdout: '',
  stderr: '/bin/sh: 1: legendary: not found\n'

```
Ubuntu 20.10.  Heroic 1.4.0 AppImage worked properly.

**Comment by flavioislima** [comment]
Duplicated of #167

**Comment by flavioislima** [comment]
Fixed on 1.5.1
